src/sagemaker_inference/errors.py

```diff
@@ -10,16 +10,16 @@
 # distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+"""This module contains user-defined exceptions."""
 from __future__ import absolute_import
 
 import textwrap
 
 
-class ClientError(Exception):
-    pass
-
-
 class UnsupportedFormatError(Exception):
+    """Exception used to indicate that an unsupported content type was provided.
+    """
```

Comment:
for one-liners I believe you're allowed to close the docstring in the same line rather than putting `"""` on its own line